src/containers/withAuthorizedAction/AuthorizationModal/utils.tsx

```diff
@@ -171,7 +185,7 @@ export function getSteps({
   authorizedContractLabel?: string
 }) {
   const requiredAllowanceAsEth = requiredAllowance
-    ? ethers.utils.formatEther(requiredAllowance)
+    ? ethers.utils.formatEther(requiredAllowance).replace(/\.0+$/, '')
```

Comment:
Is this a way to show the user without decimals what the amount that is required for the allowance? If it has decimals, meaning that the required allowance is greater than its integer value, shouldn't we ask for a greater allowance that will cover the decimal part?